mmf/modules/hf_layers.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import math
+from typing import Optional
+
+import torch
+from torch import Tensor, nn
+from transformers.modeling_bert import (
+    BertAttention,
+    BertIntermediate,
+    BertOutput,
+    BertSelfAttention,
+    BertSelfOutput,
+)
+
+
+class BertSelfAttentionJit(BertSelfAttention):
+    """
+    Torchscriptable version of `BertSelfAttention` from Huggingface transformers v2.3.0
```

Comment:
Is there any source for these modifications or are these novel?

If these are novel, it would good to delineate what exactly and why was changed for future users so they can easily adapt other models for this.

Something very similar has been in works at https://github.com/huggingface/transformers/pull/6846

Reply:
These are novel modifications. I will update in the comment what has changed for each of these. The huggingface PR isn't merging it seems.